microsoft-edge/progressive-web-apps-chromium/index.md

```diff
@@ -113,123 +129,60 @@ keywords: progressive web apps, PWA, Edge, JavaScript, Windows, UWP, Microsoft S
+While most consumers have migrated to mobile-first ways of consuming content, businesses still mostly operate on desktop.
+Building your desktop app as a PWA allows you to make use of all the benefits that PWAs come with while targeting those users.
 
-*   Your app is built on a standards-based web platform.
-*   Allows your users to install your app directly from the browser.
-*   Allows your users to install your app without a Store-based deployment or registration.
+With many new web capabilities and new features in Microsoft Edge and Windows, there are great opportunities to build immersive experiences that deeply integrate with the desktop operating system and feel native. Only using web technologies also ensures that your app will work in other browsers, operating systems and devices.
 
-Desktop PWAs are supported on any of the platforms Microsoft Edge \(Chromium\) is [available](https://www.microsoft.com/edge). The following benefits are included.
+You might not have to use native technologies at all as the web has grown capabilities to deal with file systems, video, audio, high-performing code, databases, USB, Bluetooth, and more.
 
-*   Apps may be installed directly from within the browser using the **Install** icon in the navigation bar.
+With a shared codebase between your website, mobile app and desktop app (across operating systems), you cut down on development cost.
 
-    :::image type="complex" source="./media/install-progressive-web-app-icon.png" alt-text="Install app flyout and icon" lightbox="./media/install-progressive-web-app-icon.png":::
-       Install app flyout and icon
-    :::image-end:::
+Your app can also be published on the [Microsoft Store][PwaMicrosoftStore], making it more familiar and trustworthy for Windows users to discover and install.
 
-*   Apps may also be installed, run, and managed from the **Settings** > **Apps** menu
+## Bridging the gap to native
 
-    :::image type="complex" source="./media/app-menus.png" alt-text="App menu items under settings" lightbox="./media/app-menus.png":::
-       App menu items under settings
-    :::image-end:::
+There are many new web capabilities built-in to Microsoft Edge that can make your app look and feel more native on Desktop and offer your users with engaging experiences.
 
-*   Web Notifications are integrated into the Windows notification system
-*   Shared cookie store with the browser profile that installed the app
-*   Access to other browser features using the **Setting and more** \(`...`\) menu including certificate validation, site permissions, tracking protection, and browser extensions
-*   Full access to [Microsoft Edge DevTools][DevtoolsProgressiveWebApps] for debugging your app
+With you PWAs, you can:
 
-> [!NOTE]
-> For more information about PWA benefits, upcoming features, and short demos, navigate to [Build 2020 PWA session][BuildVideo].
+*   Handle files.
```

Comment:
suggest semicolons over periods (with the exception of the last item) or no punctuation at all.

Reply:
I think the Microsoft Writing Style Guide actually advises to use periods in this case. If all items were very short (2 or 3 words each), then the guide suggests to drop punctuation, but this isn't the case here.